components/speedreader/common/BUILD.gn

```diff
@@ -0,0 +1,13 @@
+source_set("common") {
```

Comment:
i mostly did it so the settings UI in `browser/ui` can link against just those instead of pulling in all the other speedreader code it doesn't use, since it only needs preference names

Reply:
I understand your purpose. Right, `//brave/browser/extensions` target will include only `common` target.
Then, `//brave/browser/extensions` and `//brave/browser/ui` are included in one browser target.
That means, all speedreader code will be included only once.
I don't have strong opinion for this. Just left to metion about our convension.
I'll leave this to other reviewers :)

Reply:
yeah there is not much sense in adding extra directories for `common` and `buildflags` in this case, but probably not a blocker.

Reply:
i reverted it, so it's back to single directory